### ISSUE TITLE
Release 22.4.0

### ISSUE THREAD

**Issue by chadwhitacre** [getsentry/self-hosted#1421]
[22.3.0](https://github.com/getsentry/self-hosted/issues/1325) | **22.4.0** | [22.5.0](https://github.com/getsentry/self-hosted/issues/1422)

- [x] [`relay`](https://github.com/getsentry/publish/issues/1010)
- [x] [`snuba`](https://github.com/getsentry/publish/issues/1012)
- [x] [`sentry`](https://github.com/getsentry/publish/issues/1011)
- [x] [`sentry-docs`](https://github.com/getsentry/publish/issues/1014)
- [x] [`self-hosted`](https://github.com/getsentry/publish/issues/1013)

**Comment by chadwhitacre** [comment]
relay snuba sentry accepted

**Comment by chadwhitacre** [comment]
expecting self-hosted and sentry-docs to kick off in ~10 minutes

**Comment by chadwhitacre** [comment]
relay ongoing, no visibility

**Comment by chadwhitacre** [comment]
Issue with Relay: https://github.com/getsentry/publish/issues/1010#issuecomment-1100274856

Reminiscent of https://github.com/getsentry/self-hosted/issues/1193#issuecomment-995220313 🤔

**Comment by chadwhitacre** [comment]
self-hosted and sentry-docs are prepped, holding off on accepting until relay is done

**Comment by chadwhitacre** [comment]
Alright what's going on, second relay run is dead in the water after an hour, still at 5 uploads.

**Comment by chadwhitacre** [comment]
relay done, down in the muck on self-hosted

**Comment by chadwhitacre** [comment]
Kind of a mess.

The `self-hosted` release is [blocked](https://github.com/getsentry/self-hosted/runs/6042713562?check_suite_focus=true#step:4:2588) on [flaky CI](https://github.com/getsentry/self-hosted/issues/1415).

My thought was to start debugging locally to see if I could find a quick fix for the CI failure ... but I have a new M1 laptop and `self-hosted` [doesn't run on it](https://github.com/getsentry/self-hosted/issues/914).

Best bet at this point still seems to be to fix the `web` CI flake and restart the release from the beginning (re-run prepare).

Could also disable docker-compose v1 checks in CI, i.e.,  drop Docker Compse v1 support for `self-hosted`. That is not a great option because I'm sure people are still using it and we will hear about it. This would still require a restart from prepare (since we would need to disable the checks in code and re-branch).

**Comment by chadwhitacre** [comment]
So then the question is can I fix the `web` flake without a local dev environment? I guess it is a longer cycle but I can debug via CI.

**Comment by chadwhitacre** [comment]
I went down the rabbit hole on https://github.com/getsentry/self-hosted/issues/1415, afaict the CI failures in `self-hosted` under Docker Compose v1 are due to no change we introduced in any component we control (`self-hosted`, `sentry`, etc.). Best guess atm is that something changed in the GitHub Actions environment (hard for me to test because `self-hosted` [doesn't run on M1](https://github.com/getsentry/self-hosted/issues/914)).

My plan is to temporarily(?) turn off the Docker Compose v1 CI checks, and restart the `self-hosted` release with a new release branch.
